Configuration.props

```diff
@@ -58,7 +58,9 @@
     <AndroidMxeInstallPrefix Condition=" '$(HostOS)' == 'Darwin' ">$(HostHomebrewPrefix)</AndroidMxeInstallPrefix>
     <AndroidSdkDirectory Condition=" '$(AndroidSdkDirectory)' == '' ">$(AndroidToolchainDirectory)\sdk</AndroidSdkDirectory>
     <AndroidNdkDirectory Condition=" '$(AndroidNdkDirectory)' == '' ">$(AndroidToolchainDirectory)\ndk</AndroidNdkDirectory>
-    <AndroidSdkCmakeDirectory>$(AndroidSdkDirectory)\cmake\3.6.4111459</AndroidSdkCmakeDirectory>
+    <AndroidCmakeVersion Condition=" '$(AndroidCmakeVersion)' == '' ">3.6.4111459</AndroidCmakeVersion>
```

Comment:
Why distinguish between `$(AndroidCmakeVersion)` and `$(AndroidCmakeVersionPath)`?

Reply:
Because Google made a naming change with cmake 3.10 - the cmake *version* no longer constitutes part of the package name. With 3.6 the package is named `cmake-3.6.4111459.zip` and is stored in the `cmake/3.6.4111459` path in the SDK.  With 3.10, the cmake version is `3.10.2`, stored in `cmake/3.10.2.4988404` path but the package name is `cmake-3.10.2.zip` so when we bump to 3.10, the `AndroidCmakeVersion` property will have the `3.10.2` value while `AndroidCmakeVersionPath` will have the `$(AndroidCmakeVersion).4988404` value.